cas-server-core/src/main/java/org/jasig/cas/logout/LogoutManagerImpl.java

```diff
@@ -277,7 +277,7 @@ public void setSingleLogoutCallbacksDisabled(final boolean singleLogoutCallbacks
          * @param url The url to send the message to
          * @param message Message to send to the url
          */
-        public LogoutHttpMessage(final URL url, final String message) {
+        LogoutHttpMessage(final URL url, final String message) {
```

Comment:
Classes that are private dont need their ctor to be marked as public. That is redundant and useless.